src/v3/src/components/PasswordRequirements/PasswordRequirementListItem.tsx

```diff
@@ -25,6 +26,7 @@ const PasswordRequirementListItem: FunctionComponent<PasswordRequirementProps> =
     <Box
       component="li"
       sx={{ marginBlockEnd: (theme) => theme.spacing(2) }}
+      color={status === 'complete' ? green[600] : undefined}
```

Comment:
Is this pulling from Odyssey correctly? The screenshot doesn't look like our Green 600.

Reply:
Yea it did look off, maybe I need to pull the value from their Tokens? Let me dig into their repo and see, they also don't re-export the colors so I could only pull from MUI. 

Will check on this and report back 🙏

Reply:
Cool, thanks! Hmm I recall hearing that Odyssey's tokens should be available through MUI's APIs – and that would be more stable – but I don't know details. 🤷‍♂️

Reply:
Yea it looks like that is still in progress: https://oktainc.atlassian.net/browse/OKTA-545598

Once completed we will need to see how to consume it and update where we are using the `Tokens` prop.